composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "twig/twig": "~1.12,>=1.12.0",
-        "symfony-cmf/testing": "~1.2,>=1.2.6",
+        "symfony-cmf/testing": "dev-phpcr_odm_1_3 as 1.3.0",
```

Comment:
we wait till the PR is merged and then just rely on `~1.3` as version constraint